s3am/Makefile

```diff
@@ -0,0 +1,25 @@
+# Definitions
+build_tool = runtime-container.DONE
+name = quay.io/ucsc_cgl/s3am
+tag = 1.0a1
```

Comment:
Ah, I see. I'm not such a fan of including the cgl-docker-lib commit hash, but I would accede if I am outnumbered.